### DISCUSSION TITLE
Take into consideration the existing issues?

### DISCUSSION THREAD

**Discussion by alensiljak** [orgzly-revived/orgzly-android-revived#2] [General]
There are tons of suggestions and issues reported in the original repo (as we are all painfully aware of). My question is what would be the best approach in regards to these.
I have several issues open that I would still like to see implemented. Due to never getting a reply I wasn't very motivated to implement them myself.
In any case - suggestions?

**Comment by markamaze** [comment]
I started a new discussion topic to address what is needed for this forked version to be successful. One of my suggestions is informing the community and one thing to do this could be going onto the original repo and leaving comments on issues and pr's to tell people to migrate here. Those comments could include telling the community to copy their issue and create a new one here if they'd like.

However, I would suggest that many of the issues on the original repo are going to be beyond the scope of importance at the initial migration phase. It seems a lot of them are feature requests, which will bog down the few developers willing to work on things. I'd suggest focusing on the transition to the new repo while including bug fixes at first. Once we get people using the new app, I'd imagine there will not be much of a problem getting issues submitted for feature requests. And if contributors are twiddling their thumbs waiting for an issue to work on, they could be directed to look at old issues from the original repo, copy it over, and begin working on it.

**Comment by colonelpanic8** [comment]
@alensiljak Feel free to file issues against this repository, and possibly link to issues filed in the original repository.
